### PR TITLE
feat: added Homepage 

### PR DESCRIPTION
**Pull request by shrushti2000**
Created basic homepage for Naaricare ecommerce app.
1.Added Header  
2.Added carousel (not made responsive yet. Also not yet added sliding effect using animations)
3. Added categories section and a small subHeader.( As of now it is made in a static way. Categories  and subHeader part will be shifted to another component after adding context)
4. Added brands and offers section
5. Added footer.
ezgif com-gif-maker (15)

### CODE REVIEW

**Review by anuj-kumary — APPROVED** [review]
Everything looks good to me. The code is readable and well-indented. The folder structure is good too.

**Review by rohit8686 — APPROVED** [review]
Nice modularity and everything looks good !

File: src/pages/HomePage/HomePage.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+import Footer from '../../components/Footer/Footer'
+
+const HomePage = () => {
+  return (
+    <>
+
+      <SubHeader />
+      <Carousel />
+      <h5 className='sub-heading'>Categories for you</h5>
+      <div class="featured-category-section flex-hz flex-wrap">
```
Comment (ivinayakg) [inline src/pages/HomePage/HomePage.js:26]: try to see if you can or cannot make the code below into a component or try using an existing component and extending its styling through CSS, will make the homepage file better in terms of readability.

**Comment by shrushti2000** [comment]
Thank you @anuj-kumary , @rohit8686 for PR review.

File: src/pages/HomePage/HomePage.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import SubHeader from '../../components/SubHeader/SubHeader'
+import Carousel from '../../components/Carousel/Carousel'
+import categoryImage1 from '../../assets/Images/skincare-2.jpg'
+import categoryImage2 from '../../assets/Images/tampons.jpg'
+import categoryImage3 from '../../assets/Images/makeup-img-2.jpg'
+import categoryImage4 from '../../assets/Images/hair-care-img.jpg'
```
Comment (ivinayakg) [inline src/pages/HomePage/HomePage.js:7]: add these images into the public folder and then access as a URL, google this 'accessing public folder files in React' you will understand what I'm talking about.

**Review by ivinayakg — COMMENTED** [review]
I really liked your file structuring its really good need a bit of work on assets and components just a bit otherwise the code is good.

File: src/pages/HomePage/HomePage.js (hunk 1 above)
Comment (shrushti2000) [reply src/pages/HomePage/HomePage.js:26]: Thanks for the review @ivinayakg . I am going to shift categories section to a seperate component after adding context.As of now I have kept that part static.

File: src/pages/HomePage/HomePage.js (hunk 2 above)
Comment (shrushti2000) [reply src/pages/HomePage/HomePage.js:7]: Thanks for the suggestion. Will surely look into this.

**Pull request merged**